pkg/occlient/occlient_test.go

```diff
@@ -613,9 +613,9 @@ func TestUpdateDCAnnotations(t *testing.T) {
 					t.Errorf("deployment Config annotations not matching with expected values, expected: %s, got %s", tt.annotations, updatedDc.Annotations)
 				}
 			} else if err == nil && tt.wantErr {
-				t.Errorf("test failed, expected: %s, got %s", "false", "true")
+				t.Error("test failed, expected: false, got true")
```

Comment:
When you see just this error it is not clear what happened.
"error was expected, but no error returned" or something like this